src/test/java/com/kiwigrid/helm/maven/plugin/UploadMojoTest.java

```diff
@@ -94,6 +94,53 @@ public void verifyHttpConnectionForChartmuseumUpload(UploadMojo uploadMojo) thro
 		assertEquals("application/gzip", contentTypeHeader);
 	}
 
+	@Test
+	public void verifyHttpConnectionForChartmuseumUploadWithCredentials(UploadMojo uploadMojo) throws IOException {
```

Comment:
This test covers only, whether the correct HTTP method and content-type was set.
From verification point-of-view it's actual equal to already existing test `verifyHttpConnectionForChartmuseumUpload`